net/dns/direct.go

```diff
@@ -183,6 +184,24 @@ func (m *directManager) readResolvFile(path string) (OSConfig, error) {
 	return readResolv(bytes.NewReader(b))
 }
 
+func (m *directManager) GetExitNodeForwardResolver() (ret []dnstype.Resolver, retErr error) {
+	for _, filename := range []string{backupConf, resolvConf} {
+		if oc, err := m.readResolvFile(filename); err == nil {
+			for _, ip := range oc.Nameservers {
+				if ip != netaddr.IPv4(100, 100, 100, 100) {
```

Comment:
No need to specialcase quad 100. If that's what the exit node uses for DNS, it's perfectly fine for peerapi to loop back into it.

Reply:
We don't want to loop back to 100.100.100.100 and get the exit node's MagicDNS names blended in if it's in a different tailnet.

We can discuss in eng meeting.